open_spiel/python/egt/alpharank.py

```diff
@@ -791,3 +791,41 @@ def compute(payoff_tables,
     print_results(payoff_tables, payoffs_are_hpt_format, rhos, rho_m, c, pi)
 
   return rhos, rho_m, pi, num_profiles, num_strats_per_population
+
+def suggest_alpha(payoff_tables):
+  """Suggests an alpha for use in alpha-rank. 
+
+  The suggested alpha is approximately the smallest possible alpha such that 
+  the ranking has 'settled out'. It is calculated as 2/min_gap_between_payoffs. 
+
+  Args:
+    payoff_tables: List of game payoff tables, one for each agent identity. Each
+      payoff_table may be either a numpy array, or a _PayoffTableInterface
+      object.
+  
+  Returns:
+    A suggested alpha.
+  """ 
+  num_strats_per_population =\
+    utils.get_num_strats_per_population(payoff_tables, False)
```

Comment:
Would you mind adding `payoffs_are_hpt_format = utils.check_payoffs_are_hpt(payoff_tables)` at the top of the function, and using this in lieu of the hard-coded False here? Thanks!